edx_rbac/utils.py

```diff
@@ -62,8 +63,12 @@ def user_has_access_via_database(user, role_name, role_assignment_class, context
         return False
 
     if context:
-        return role_assignment.get_context() in (context, ALL_ACCESS_CONTEXT)
-
+        context_in_database = role_assignment.get_context()
```

Comment:
can you update the comment for this function to describe how context is evaluated? it took me a couple reads to understand what the logic was and why it made sense.

Reply:
Done. Thanks

Reply:
Done